pkg/endpoint/endpoint.go

```diff
@@ -579,6 +603,9 @@ func CreateIngressEndpoint(owner regeneration.Owner, policyGetter policyRepoGett
 	ep.DatapathConfiguration = NewDatapathConfiguration()
 
 	ep.isIngress = true
+	ep.properties[PropertySkipBPFPolicy] = true
+	ep.properties[PropertyWithouteBPFDatapath] = true
```

Comment:
Is this change intentional? Can you add a comment?
How was this handled before this change?

Reply:
> Is this change intentional? Can you add a comment?

It is intentional, I forgot to add a comment.

This was handled before with the `ep.isIngress = true`.